backend/apps/volontulo/factories.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+"""
+.. module:: factories
+"""
+
+import factory
+from django.contrib.auth import get_user_model
+from faker import Faker
+
+from apps.volontulo.models import UserProfile
+
+fake = Faker(locale='pl_PL')
+
+User = get_user_model()
+
+
+class UserProfileFactory(factory.DjangoModelFactory):
+    """Factory for user profile."""
+
+    class Meta:  # pylint: disable=C0111
+        model = UserProfile
+
+
+class UserFactory(factory.DjangoModelFactory):
+    """Factory for User."""
+
+    class Meta:  # pylint: disable=C0111
+        model = User
+
+    first_name = fake.first_name()
+    last_name = fake.last_name()
+    username = factory.LazyAttribute(
```

Comment:
in `views/auth.y` we have: 
```
        username = request.POST.get('email')
        password = request.POST.get('password')
```
can we use email also as username instead of `firstname.lastname`?